semgrep_output_v1.atd

```diff
@@ -516,28 +516,30 @@ type ecosystem
   | Mix <json name="mix">
   | Hex <json name="hex">
   | Opam <json name="opam">
-]
+] <ocaml repr="classic">
 
-type transitivity
+type dependency_kind
```

Comment:
I am a little hesitant about this change, I think we might introduce additional categories like dev dependencies that might make this name confusing. But I guess this is changeable in the future so not a big deal

Reply:
I agree. I hesitated too, but seeing some out.Transitivity(out.Transitive) convinced me :)